var/spack/repos/builtin/packages/openblas/package.py

```diff
@@ -6,15 +6,20 @@ class Openblas(Package):
     homepage = "http://www.openblas.net"
     url      = "http://github.com/xianyi/OpenBLAS/archive/v0.2.15.tar.gz"
 
-    version('0.2.16', 'fef46ab92463bdbb1479dcec594ef6dc')
+    # version('0.2.16', 'fef46ab92463bdbb1479dcec594ef6dc') # blacklist: miss _dggsvd_ and _sggsvd_ at least on OSX
     version('0.2.15', 'b1190f3d3471685f17cfd1ec1d252ac9')
 
+    variant('tests', default=True, description='Run small tests')
```

Comment:
We might want to leave this out for now and wait until a consensus is reached in #169.